_pages/404.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: not_found
+title: This page is no longer available.
+permalink: /404.html
+sitemap: false
+---
+
+<main class="site-wrapper">
+  <div class="site-wrapper-inner">
+    <div class="cover-container">
+      <div class="masthead clearfix">
+        <div class="inner">
+          <a href="/"><img src="{{ '/assets/img/logo-white.svg' | prepend: site.baseurl}}" width="150" alt="Login.gov logo" class='masthead-brand'/></a>
+        </div>
+      </div>
+      <div class="inner cover">
+        <h1>
+          This page is no longer available.
```

Comment:
Nitpick: I know you copied this from another of our sites, but this may not be precise.

The page may be available at another URL. Or, the user may have entered an arbitrary URL that never linked to a page.
My suggestion is "The page you are looking for may have moved. Please check the URL." Though Karen would be a better person to ask.

Reply:
Yeah I agree that the language could be more precise. I guess the question is whether we want consistency in the language with the brochure site.

Reply:
I agree with AJ. More precise language wins over consistency.